Source/WebKit/UIProcess/Network/CustomProtocols/LegacyCustomProtocolManagerProxy.h

```diff
@@ -25,6 +25,7 @@
 
 #pragma once
 
+#include "CacheStoragePolicy.h"
```

Comment:
This can probably be a forward declaration instead.
enum class CacheStoragePolicy : uint8_t;

Reply:
Done.